### ISSUE TITLE
15 duplicate classifications from a non-logged in user in the span of 5 minutes

### ISSUE THREAD

**Issue by mschwamb** [zooniverse/Panoptes#1918]
I'm writing the Planet Four: Terrains paper and looking at the number of classifications where after removing duplicates there were significantly less than 20 independent clicks. 

I saw this:

<img width="661" alt="screen shot 2016-08-21 at 11 53 34 am" src="https://cloud.githubusercontent.com/assets/753325/17844930/e14004b0-67d9-11e6-910e-3b517427e031.png">

14 classifications in the span of something like 5 minutes came from one IP address/non-logged in session. This is an extreme case, but it seems like this should be prevented by the backend. I didn't find many subjects as extreme as this, but I thought I'd mention it.

**Comment by mschwamb** [comment]
I guess it should be a school class classifying, but it seems odd that they'd all get this subject.

**Comment by camallen** [comment]
I don't think the backend should stop these, you mentioned school but also what about kiosk computers in a museum / tablets at an outreach event over a shared LAN connection (ip appears the same). 

I agree the created at stamps look a little close together but this could be viable. I think this is a problem for aggregation. Out of interest are the classifications junk or mildly in agreement with others you trust?

**Comment by marten** [comment]
Noting here that all of those classifications have a different session identifier in the metadata. Which means they were either reloads (but then how did they get the same subject again?) or genuinely a bunch of computers behind a NAT (but then why did they all get the same subject?).

Both options are really weird 🎱

**Comment by mschwamb** [comment]
@marten right that's what I find odd. If it's behind a NAT, I find it odd they'd all get the same subject.  

@camallen the classifications are junk.

**Comment by camallen** [comment]
@mschwamb how prevalent in your data is this issue?

**Comment by mschwamb** [comment]
@camallen I treat ip address as best I can as 'individuals' for weighting and have to delete 13 of those entries from use. There are probably other things I could do. It doesn't seem to be many. I've got 15 subjects like this. It just looked really odd so I figured it was worth mentioning.

I don't get session ids in the csv output so I couldn't tell if these were in the same sitdown session.  The only thing I can think of was there wasn't that many live subjects left live, but still seems odd.

Feel free to close the ticket as you see fit.
